core/CliMulti/RequestCommand.php

```diff
@@ -89,7 +96,7 @@ protected function initHostAndQueryString(InputInterface $input)
         Url::setHost($hostname);
 
         $query = $input->getArgument('url-query');
-        $query = UrlHelper::getArrayFromQueryString($query);
+        $query = UrlHelper::getArrayFromQueryString($query); // NOTE: this method can create the StaticContainer now
```

Comment:
RequestCommand deletes the container and recreates it w/ a different host + config. After setting the host and config it checks for the test environment, however, if getArrayFromQueryString is called before the Config for the test environment is setup, the test overrides are ignored. It's all very confusing and very annoying.